python/cuml/tests/test_kernel_ridge.py

```diff
@@ -178,6 +178,7 @@ def array_strategy(draw):
 
 @given(kernel_arg_strategy(), array_strategy())
 @settings(deadline=None)
+@pytest.skip('https://github.com/rapidsai/cuml/issues/5177')
```

Comment:
My suggestion was wrong, sorry for that, it should be `@pytest.mark.skip` (which I always confuse...)